src/ape/managers/chain.py

```diff
@@ -359,9 +360,22 @@ def base_fee(self) -> int:
     def pending_timestamp(self) -> int:
         """
         The current epoch time of the chain, as an ``int``.
+        Set the timestamp to fast-forward time into the future.
+
+        Usage example::
+
+            from ape import chain
+
+            chain.pending_timestamp += 3600
         """
 
-        return int(time.time())
+        return int(time.time() + self._time_offset)
```

Comment:
Just noticed that you could ask for `self.provider.get_block("pending").timestamp` to get this value from the provider

Reply:
Oh nice!